doc/source/metadata/Working-with-Metadata.rst

```diff
@@ -0,0 +1,295 @@
+
+General post-processing
+***********************
+
+Post-processing, such as neighbourhood processing,
+does not change the underlying nature of the parameter.
+Such plugins can therefore copy a cube and modify specific metadata
+(coordinates, attributes), but can safely inherit all other existing metadata
+as it will remain correct.  Most ``PostProcessingPlugin`` instances
+will copy cubes in this way.
+
+New parameters
+**************
+
+When creating new parameters, developers **should not** 
+copy an input cube directly,
+but should make use of the utility to create new parameter cubes.
+This makes use of ONLY the coordinates from a template cube,
+and adds specific attributes and cell methods as required.
+Developers should take care to:
+
+* Provide a correct template cube, i.e. by removing any scalar coordinates
+  that are not relevant to the new parameter
+* Provide suitable mandatory attributes
+  (``source``, ``title`` and ``institution``).
+  These should usually be derived using the “generate mandatory attributes”
+  function from **all** input parameters.
+  (E.g. in weather symbols, all the different fields - 
+  precipitation, cloud, lightning, etc - should be read into this function.)
+* **NOT** simply pass in all attributes from the template cube,
+  as these may be inappropriate to the new parameter
+* **NEVER** to copy cell methods from the template cube,
+  as these will be inappropriate to the new parameter
+* Consider whether this parameter may be needed as a level 3 blended field,
+  or as input to weather symbols.
+  If so, it will need an option to inherit a model ID attribute.
+
+Minimal metadata
+****************
+
+The IMPROVER metadata principles include that the metadata should be
+the **minimum** required to fully describe the parameter,
+and that the metadata should be **correct**.
+The main setting where developers need to understand this is in
+creating new parameters.
+Practical implications include:
+
+Positive selection
+    Choosing a specific set of attributes to **include**,
+    rather than a specific set to **exclude**.
+    This means a new parameter plugin does not inherit anything
+    unexpected by default, which may not be “correct” for the new parameter.
+
+Clear internal responsibility
+    Defining within the plugin **all** new attributes and / or cell methods 
+    which are required to describe this new dataset.
+
+The **only** case for a plugin not taking full responsibility for metadata
+is if organisation-specific details - such as the name of the model ID attribute
+- need to be passed in via the command line.
+Even in these cases, the plugin should take as much responsibility as possible,
+requiring minimal information from the user to inform metadata updates.
+(For example, in the model ID attribute case,
+the user is required to provide the name of the attribute from which to read
+model information, rather than a ``name: value`` pairing
+to be directly applied.)
```

Comment:
```suggestion
to be directly applied).
```

Reply:
I'm not sute that I agree with the placing of the full stop outside the bracket given that the sentence is fully contained within the brackets, so I've side-stepped the issue by just removing the brackets completely.